### DISCUSSION TITLE
Possible to make IDE "Go to definition" work?

### DISCUSSION THREAD

**Discussion by WesleyYue** [pmndrs/zustand#1576] [Q&A]
Is there a way to make "go to definition" work for zustand? Let's say you have the following:

```
const increasePopulation = useStore((state) => state.increasePopulation)
```

- Trying to go-to-definition on useStore will bring you to the top of the `create(...)` call
- doing go-to-definition on `increasePopulation` of `state.increasePopulation` will take you to the type definition for `increasePopulation` if you are using typescript. 

Is there some syntax I can use to make it so that go-to-definition will go to where increasePopulation is actually defined within the `create({..., increasePopluation: ...)` call?

**Comment by dai-shi** [comment]
I'm not sure if it's possible. Would be nice though.

FWIW, a completely different approach is: https://github.com/pmndrs/zustand/blob/44b23382f0ff4a29f2c9deb84150ddf34338c200/docs/guides/practice-with-no-store-actions.md
